ocaml/lambda/translcore.ml

```diff
@@ -1041,22 +1049,23 @@ and transl_apply ~scopes
             | Alloc_local -> false
             | Alloc_heap -> true
           in
+          (* CR ncourant: need layout of the omitted parameter *)
```

Comment:
It's likely we'll change the typed tree to store layout information in `apply_arg`s when we add front-end support for non-value parameters, FYI.